app/views/new_administrateur/procedures/_publication_form.html.haml

```diff
@@ -0,0 +1,41 @@
+.card.mb-4
+  %h4.card-title Adresse de la démarche
```

Comment:
Dans les nice to have ce serait bien de remplacer les h4 par des h2 (et ajuster le rendu), en l'état la structure html passe de h1 à h4